src/main/java/mate/academy/springintro/SpringIntroApplication.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.springintro;
+
+import java.math.BigDecimal;
+import mate.academy.springintro.model.Book;
+import mate.academy.springintro.service.BookService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class SpringIntroApplication implements CommandLineRunner {
```

Comment:
Where is your controller with task implementation?

Book Endpoints:

GET: /api/books (Retrieve book catalog) (should be done in the previous PR)
GET: /api/books/{id} (Retrieve book details) (should be done in the previous PR)
POST: /api/books (Create a new book) (should be done in the previous PR)
PUT: /api/books/{id} (Update a specific book)